inst/rmarkdown/templates/oup/skeleton/skeleton.Rmd

```diff
@@ -23,69 +23,75 @@ author:
         - 2
 address:
   - code: ABC
-    address: Address ABC
+    department: Department of Biology
+    organization: University of Idaho
+    street: 123 Main Street
+    #postcode: 
+    city: Boise
+    state: Idaho 07458 # Mix and match elements to get proper address
+    country: United States
   - code: DEF
-    address: Address DEF
+    address: Address in one element deprecated, but works
   - code: Another University
     address: Address for Another University
 footnote:
   - code: 1
     text: "Equal contribution"
   - code: 2
-    text: "Current email address: \\href{mailto:cat@example.com}{cat@example.com}"
-abstract: |
-  This is the abstract.
-
-  It consists of two paragraphs.
+    text: "Current email address: [cat@example.com](mailto:cat@example.com)"
+abstract:
+  - "This is the first paragraph of the abstract. It has several sentences making it go over several lines. For this it needs to have a lot of text." 
+  - "This is the second paragraph."
+  - "**Motivation:** You can also have some paragraphs start with bold face."
+keywords:
+  - keyword1
+  - keyword2
+  - keyword3
+#boxedlist: # Not fully compatible with all document formats
+#  - First point
+#  - Second point
+competing_interests: |
+  There are no competing interest.
+author_contributions: |
+  AA did all the work. The others are just freeloaders.
 acknowledgements: |
   This is an acknowledgement.
 
   It consists of two paragraphs.
-keywords:
-  - key
-  - dictionary
-  - word
-#fontsize: 12pt
-#spacing: halfline # could also be oneline
 #classoptions:
-#  - endnotes
-bibliography: mybibfile.bib
+#  - namedate # Could also be here...
+bibliography: mybibfile.bib # Includes refs in OUP example template
 #csl: mycitationstyle.csl
-#link-citations: yes
-output: rticles::oup_article
+output:
+  rticles::oup_article:
+    keep_tex: TRUE
+    journal: "Journal Name"
+    #document_style: "contemporary" # Can be contemporary, modern, traditional; default "contemporary"
+    #papersize: "large" # Can be large, medium, small; default large
+    #citation_package: "default" # Uncomment if using a CSL biblio style; default "natbib"
+    namedate: TRUE # Set FALSE to use numeric refs; Default FALSE
+    #number_sections: FALSE # Uncomment to not number sections; default TRUE
+    #onecolumn: TRUE # Uncomment for one column format; default FALSE
 #urlcolor: orange
 #linkcolor: green
 #citecolor: red
 #header-includes:
 #  - \usepackage[nomarkers,tablesfirst]{endfloat} # For figures and tables at end
-#  - \usepackage{lineno} # For line numbering
-#  - \linenumbers # For line numbering
+#  - \usepackage{lineno}\linenumbers # For line numbering
```

Comment:
I think you put that already int the template using  using a variable. 
Maybe this option should be shown ? instead of header includes.

Reply:
I explain above why I chose `header-includes` instead of showing the option. But I am open to rethinking things provided a good way of passing options to `$number_lines$` is found.

Reply:
I am just saying here that I think we need one of the other. 

I am fine with showing what to pass in header-includes, but in that case I think we should not add the line in the template. 

It is the duplication that bothers me.

Reply:
I have implemented the "format function" solution on the branch linked to PR #431. I did this in two steps, first implementing it at the base YAML level using `numberlines` and then moving it into the format function in commit `5f143751272242eb975f70cf8adf299f67fbd2ab`. If ever it is really a problem to put this option into the format function, then we can just undo this single commit to get back to the base level configuration.